tools/classifier/sample.env

```diff
@@ -5,3 +5,7 @@ TOOL_DATA_DIR=../data_dir
 
 X2TEXT_HOST=http://unstract-x2text-service
 X2TEXT_PORT=3004
+
+EXECUTION_DATA_DIR=<execution_dir_path_with_bucket>
```

Comment:
NIT: I suggest to add a comment above these envs indicating that its for remote storage. Atleast the other 2 envs have `FS` in them but just `EXECUTION_DATA_DIR` will be confusing when we already have a lot of other similar named envs

Reply:
Updated env with comment. All other DIR envs will be removed once FS is implemented.